t3/settings/submit_zeus.py

```diff
@@ -0,0 +1,81 @@
+"""
+Submit scripts
+"""
+
+# Submission scripts stored as a dictionary with software as the primary key.
+submit_scripts = {
+#     'rmg': """#!/bin/bash -l
```

Comment:
can remove the commented out section, this is an example for a Slurm server, it's enough to have it in the general submit.py file